cedar-policy-formatter/src/pprint/fmt.rs

```diff
@@ -69,7 +77,7 @@ fn soundness_check(ps: &str, ast: &PolicySet) -> Result<()> {
                 .eq_shape(p.non_scope_constraints()))
         {
             return Err(miette!(
-                "policies differ in meaning or annotations:\noriginal: {p}\nformatted: {f_p}"
+                "policies differ in policy ids or meaning or annotations:\noriginal: {p}\nformatted: {f_p}"
```

Comment:
Nit: probably easier for debugging if you separate these cases.

Reply:
I think I'll leave it to future work